includes/common.php

```diff
@@ -1510,14 +1510,21 @@ function clean($value)
  * @param $value
  * @return string
  */
-function display($value)
+function display($value, $purifier_config = array())
 {
     /** @var HTMLPurifier $purifier */
     global $config, $purifier;
+
+    if (empty($purifier_config)) {
```

Comment:
Done, it's so that we can pass on purifier config options at which point we don't want to have munged the html - see html/includes/common/notes.inc.php as an example.